### ISSUE TITLE
Programming language icons

### ISSUE THREAD

**Issue by Camilot55** [giuspen/cherrytree#1588]
Hi, Giuseppe. There is a suggestion to insert programming language icons instead of these boring brackets. See screen
![Безымянный1](https://user-images.githubusercontent.com/76065578/113444463-6383cd80-940d-11eb-9f51-d0d98be9a39f.png)

**Comment by Camilot55** [comment]
![Безымянный1](https://user-images.githubusercontent.com/76065578/113444683-cffecc80-940d-11eb-96d3-576057920a8c.png)

**Comment by giuspen** [comment]
Hi Victor sure the brackets are the default when the dedicated icon is missing but we want to find icons for more programming languages, I'll have a look and see if I can cover few more.

**Comment by Camilot55** [comment]
Hi, Giuseppe. I've already found almost everything, I can drop the archive

**Comment by giuspen** [comment]
Hi Viktor yes please if you provide the archive I'll sort it out. Note that the countries icons you provided were PNG embedded into SVGs rather than effective SVGs so I replaced with better qualities free icons set I have which have pretty much all possible flags. It would be better if you find real PNGs but if not anyway I'll see if I can find a better alternative. Careful with the licences then fetching from the internet. Cheers!

**Comment by Camilot55** [comment]
Yes, unfortunately the icon was in PNG, I converted it to SVG. Apparently not a good converter.

**Comment by Camilot55** [comment]
I downloaded it from the site
https://icon-icons.com/ru/%D0%B7%D0%BD%D0%B0%D1%87%D0%BE%D0%BA/%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D1%83%D0%B5%D1%82%D1%81%D1%8F-CSharp-%D0%BE%D1%80%D0%B8%D0%B3%D0%B8%D0%BD%D0%B0%D0%BB-%D0%BB%D0%BE%D0%B3%D0%BE%D1%82%D0%B8%D0%BF/146578

**Comment by giuspen** [comment]
Скачать SVG отлично :) it's written free for commercial use, should be good!

**Comment by giuspen** [comment]
The SVG is vectorial, it's like the source code while PNG is the binary, I don't think it is possible to get a real SVG out of a PNG while on the contrary from an SVG you can generate all sizes of PNGs
